pkg/apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -172,13 +172,20 @@ type BackendSpec struct {
 	// +optional
 	RedisImage *string `json:"redisImage,omitempty"`
 	// +optional
+	RedisPersistentVolumeClaimSpec *BackendRedisPersistentVolumeClaimSpec `json:"redisPersistentVolumeClaim,omitempty"`
+	// +optional
 	ListenerSpec *BackendListenerSpec `json:"listenerSpec,omitempty"`
 	// +optional
 	WorkerSpec *BackendWorkerSpec `json:"workerSpec,omitempty"`
 	// +optional
 	CronSpec *BackendCronSpec `json:"cronSpec,omitempty"`
 }
 
+type BackendRedisPersistentVolumeClaimSpec struct {
```

Comment:
why not reusable `PersistentVolumeClaimSpec`?

Reply:
See comment below